AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.m

```diff
@@ -9,6 +9,16 @@
 
 @interface MSAuthTokenContext ()
 
+/**
+ * Authorization token cached value.
+ */
+@property(nullable, nonatomic, copy) NSString *authToken;
```

Comment:
Can it be `atomic`?

Reply:
No. `Writable atomic property cannot pair a synthesized setter with a user defined getter`
Plus, it isn't required there: it's accessed only from `@synchronized` block.

Reply:
Overall, it doesn't need to have customized getter/setter. The getter only synchronize the property, there is no custom setter anymore.

Reply:
There is a setter. It's generated, do things like `copy` and is used only as a private setter.

Reply:
It didn’t mean atomic will solve the synchronization block in getter. I didn’t see any reason of getting authToken requires locking the whole instance. What’s why I thought getter can be removed, then authToken property can become an atomic property. authToken seems it can be used as soon as it is set. If it is not correct then it totally makes sense.

Reply:
Seems everyone wants to lock the instance when `authToken` is set. Let's keep it.